### PR TITLE
Rename `or_lesser` range property hint to `or_less`

### PR DESCRIPTION
**Pull request by Calinou**
"less" should be used for quantity, rather than "lesser".

Existing scripts that use `or_lesser` in `_get_property_list()` will need to be updated to account for this change.

See https://github.com/godotengine/godot/issues/54161#issuecomment-1035961636.

### CODE REVIEW

File: editor/editor_properties.cpp [hunk 1]
```diff
@@ -3376,7 +3376,7 @@ static EditorPropertyRangeHint _parse_range_hint(PropertyHint p_hint, const Stri
 				degrees = true;
 			} else if (slice == "or_greater") {
 				hint.greater = true;
-			} else if (slice == "or_lesser") {
+			} else if (slice == "or_less") {
 				hint.lesser = true;
```
Comment (akien-mga) [inline editor/editor_properties.cpp:3380]: Should be renamed too. Maybe rename both to include `or_` or `allow_`.

**Review by YuriSizov — APPROVED** [review]
Needs a rebase

**Comment by akien-mga** [comment]
Bump :)

**Comment by Calinou** [comment]
Rebased and tested again, it works as expected.

**Pull request merged**